### PR TITLE
rl-90/liveness Liveness:

### PR DESCRIPTION
**Pull request by alshund**
- add liveness endpoints open api definition without components descriptions

### CODE REVIEW

File: index.yml [hunk 1]
```diff
@@ -8,3 +8,5 @@ paths:
     $ref: './comparison.yml#/paths/~1compare'
   /detect:
     $ref: './detect.yml#/paths/~1detect'
+  /liveness:
```
Comment (hleb-albau) [inline index.yml:11]: other liveness paths ?

File: liveness.yml [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/LivenessResultItem"
+        400:
+          $ref: "./common.yml#/components/responses/BadRequest"
+        403:
+          $ref: "./common.yml#/components/responses/BadLicense"
+
+/liveness/depth:
```
Comment (hleb-albau) [inline liveness.yml:27]: forgot space?

File: liveness.yml [hunk 3]
```diff
@@ -0,0 +1,142 @@
+openapi: 3.0.3
+paths:
+
+ /liveness:
+    post:
+      tags:
+        - liveness
+      summary: ""
```
Comment (hleb-albau) [inline liveness.yml:8]: empty summary

File: liveness.yml [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+      operationId: check_video_liveness
+      requestBody:
+        content:
+          application/octet-stream:
+            schema:
+              $ref: "./common.yml#/components/schemas/Bytes"
+      responses:
+        200:
+          description: ""
```
Comment (hleb-albau) [inline liveness.yml:17]: empty desc

**Review by hleb-albau — CHANGES_REQUESTED** [review]
fill all desc and summaries

**Comment by alshund** [comment]
@dimkasss можешь посмотреть descriptions? Мб ты можешь что-то более точнее описать?

File: liveness.yml [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+      type: object
+      required:
+        - face
+        - livenessStatus
+      properties:
+        face:
+          $ref: "./common.yml#/components/schemas/ImageData"
+        livenessStatus:
+          $ref: "#/components/schemas/LivenessStatus"
+        code:
```
Comment (ikliashchou) [inline liveness.yml:127]: should be enum for this code value

**Comment by ikliashchou** [comment]
There is an enum for error codes from core lib. We need to add it to API desc as FaceRErrorCodes

enum eFaceRProcessorErrorCodes
{
   FACER_OK = 0,
   FACER_NO_LICENSE = 200,
   FACER_IS_NOT_INITIALIZED = 201,
   FACER_COMMAND_IS_NOT_SUPPORTED = 202,
   FACER_COMMAND_PARAMS_READ_ERROR = 203,
   FACER_SEARCHER_INDEX_FILE_NOT_EXIST = 204,
   FACER_SEARCHER_INDEX_FILE_READ_ERROR = 205,
   FACER_SEARCHER_INDEX_FILE_DECRYPTION_ERROR = 206,
   FACER_SEARCHER_INDEX_LOAD_ERROR = 207,
   FACER_SEARCHER_WRONG_DESCRIPTOR_VERSION = 208,
   FACER_SEARCHER_INDEX_FILE_ENCRYPTION_ERROR = 209,
   FACER_SEARCHER_INDEX_FILE_SAVE_ERROR = 210,
   FACER_SEARCHER_IMAGE_DECODE_ERROR = 211,
   FACER_SEARCHER_ID_EXISTS_IN_INDEX = 212,
   FACER_SEARCHER_ID_NOT_IN_INDEX = 213,
   FACER_SEARCHER_NO_FACES = 214,
   FACER_SEARCHER_DESCRIPTOR_EXTRACTOR_ERROR = 215,
   FACER_SEARCHER_METADATA_READ_ERROR = 216,
   FACER_SEARCHER_NO_SEARCH_LICENSE = 217,
   FACER_SEARCHER_DESCRIPTOR_DECODE_ERROR = 218,
   FACER_SEARCHER_DESCRIPTOR_WRONG_SIZE = 219,
   FACER_SEARCHER_ZIP_IMAGE_READ_ERROR = 220,
   FACER_SEARCHER_ZIP_META_READ_ERROR = 221,
   FACER_SEARCHER_SIMILAR_PHOTOS_EXIST = 222,
   FACER_SEARCHER_PERSON_ZIP_READ_ERROR = 223,
   FACER_LESS_THAN_TWO_IMAGES_IN_REQUEST = 224,
   FACER_SEARCHER_WRONG_ID = 225,
   FACER_SEARCHER_ID_EXISTS_IN_DELETED = 226,
   FACER_VIDEO_DECODE_ERROR = 227,
   FACER_NOT_ENOUGH_FRAMES = 228,
   FACER_OUTPUT_IS_NOT_DEFINED = 229
};

**Review by ikliashchou — APPROVED** [review]

**Pull request merged**